content/latest/faq/compatibility.md

```diff
@@ -13,11 +13,18 @@ isTocNested: true
 showAsideToc: true
```

Comment:
This is more for the future... the current structure has "FAQ" -> "Other FAQs -> ...

Maybe we can drop the "Other FAQs" sublevel. Also, might be good to have separate sections for each API under API compatibility.